osbenchmark/worker_coordinator/worker_coordinator.py

```diff
@@ -1449,7 +1449,17 @@ async def run(self):
         def os_clients(all_hosts, all_client_options):
             opensearch = {}
             for cluster_name, cluster_hosts in all_hosts.items():
-                opensearch[cluster_name] = client.OsClientFactory(cluster_hosts, all_client_options[cluster_name]).create_async()
+                if "connection_class" in all_client_options["default"]:
```

Comment:
Consider changing the user flag to `backend-client` or `client`, since this flag will continue to be used for future clients like a Rust client, C++ client, etc.

Reply:
I will rename the user flag to backend-client